src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.service.exception.DataProcessingException;
 
 public class ActorServiceImpl implements ActorService {
+
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElseThrow(
+                () -> new DataProcessingException("Not found actor with id " + id));
```

Comment:
```suggestion
                () -> new RuntimeException("Not found actor with id " + id));
```
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/20326858/8184abd8-dfe1-4bbd-acbe-4916df35e71a)